backend/StreamServerApp/database_utils.py

```diff
@@ -46,13 +47,15 @@ def pretty(d, indent=0):
             print('\t' * (indent+1) + str(value))
 
 
-def populate_db_from_local_folder(base_path, remote_url):
+def populate_db_from_local_folder(base_path, remote_url, keep_files=False):
     """ # create all the videos infos in the database
         Args:
         remote_url: baseurl for video access on the server
         base_path: Local Folder where the videos are stored
+        keep_files: Keep original files in case of convertion
```

Comment:
This affects only video files (I'll clarify that). I think it depends if you have limited space on your server, there is no need to keep a duplicate of your former mkv files, but this can be intrusive.